aat/src/aat/resources/features/F-102 - TTL Base Complex Type/F-102_Test_Complex_Organisation.td.json

```diff
@@ -0,0 +1,25 @@
+{
+  "_guid_": "F-102_Test_Complex_Organisation",
```

Comment:
This file is misnamed and may not be needed if `Get_CaseType` can be used in same way as with `S-100.3_Get_CaseType.td.json`

Reply:
Resolved in 5a66d367471da7007f70397a0e432852e0826c92